packages/ui/src/forum/modals/CreateThreadModal/machine.ts

```diff
@@ -41,8 +41,6 @@ export type CreateThreadEvent =
   | { type: 'PASS'; memberId: string; categoryId: string; controllerAccount: Account }
   | { type: 'NEXT' }
   | { type: 'BACK' }
-  | { type: 'SET_TOPIC'; topic: string }
```

Comment:
Would be good to delete `topic` and `description` from context in this file keep typing correct on machine usage